src/payment/strategies/offsite/offsite-payment-strategy.ts

```diff
@@ -32,8 +33,12 @@ export default class OffsitePaymentStrategy implements PaymentStrategy {
                     payment.methodId,
                     payment.gatewayId,
                     instrumentId,
-                    shouldSaveInstrument))
-            );
+                    shouldSaveInstrument,
+                    {
+                        ...getBrowserInfo(),
+                        time_zone: new Date().toLocaleTimeString('en-us', {timeZoneName: 'short'}).split(' ')[2],
```

Comment:
Please, add `time_zone` to `getBrowserInfo`. There's no need to only generate this for this particular case.

Also, it seems like `timeZoneName` might not be supported in `ie`. Can you please check:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString

<img width="618" alt="image" src="https://user-images.githubusercontent.com/4542735/76935913-1938c700-6946-11ea-8a81-e9419f9c0bf1.png">